features/admin.server-configurations.v1/pages/connector-edit-page.tsx

```diff
@@ -413,8 +397,7 @@ export const ConnectorEditPage: FunctionComponent<ConnectorEditPageInterface> =
                 </>);
             case ServerConfigurationsConstants.ACCOUNT_RECOVERY_CONNECTOR_ID:
                 return type === "username"
-                    ? "Enable self-service username recovery for users on the login page." +
-                        " The user will receive a username reset link via email upon request."
+                    ? "Enable self-service username recovery for users on the login page."
```

Comment:
Is there any specifc reason not to use i18n for these strings?

Reply:
I have only update the previous content and this is moved to i18n configs by https://github.com/wso2/identity-apps/pull/7004/commits/59d040047dd8b21efd511e781c167e8ebd65b64f